octoprint_mrbeam/software_update_information.py

```diff
@@ -32,7 +37,7 @@ def get_update_information(self):
 		set_info_rpiws281x(self, tier)
 #		set_info_testplugin(self, tier) # See function definition for more details
 
-	# _logger(self).debug("MrBeam Plugin provides this config (might be overridden by settings!):\n%s", yaml.dump(sw_update_config, width=50000).strip())
+	_logger(self).debug("MrBeam Plugin provides this config (might be overridden by settings!):\n%s", yaml.dump(sw_update_config, width=50000).strip())
```

Comment:
Please uncomment this line again. (So that it is a comment and doesn't get executed) It's output is useful for debugging but too misleading to keep it in there.
It's misleading because anything in confi.yamel overrides these values but you wouldn't see it in this log output.

Reply:
done